### PR TITLE
SEO pages (#9761, #9759, #9760)

### LINKED ISSUE #9759: Migrate translate blog post to www
## Description

Migrate translate blog post https://blog.mozilla.org/firefox/translate-the-web/ to www with updated copy.

Design Template: same as https://www.mozilla.org/en-US/firefox/browsers/windows-64-bit/

Copy: https://docs.google.com/document/d/1IjNb9oGR4G4ljKUXppWc0N6GlJkHuBd3QqT9wleFB-c/edit

Url: /firefox/features/translate/

Subnav like all /features/ pages

---

#### Success Criteria
- [x] Page is coded
- [ ] Blog article is redirected 301 to the new url on www
- [x] Add page to /more
- [x] Add page to /features

### LINKED ISSUE #9760: Migrate Password blogpost to www
## Description

Migrate password blog post https://blog.mozilla.org/internetcitizen/2017/01/25/better-password-security/ to www with updated copy.

Design Template: same as https://www.mozilla.org/en-US/firefox/browsers/windows-64-bit/

Copy: https://docs.google.com/document/d/1cKki_xH0AeZEcBOD7sqw_O0XpKazYRkUA70i-L0v2C4/edit#

Url: /firefox/privacy/safe-passwords/

Subnav like all /firefox/privacy/ pages, this page added to the subnav

---

#### Success Criteria
- [x] Page is coded
- [ ] Blog article is redirected 301 to the new url on www
- [x] Add page to /more
- [x] Link page in subnav of firefox/privacy/

### LINKED ISSUE #9761: Migrate fingerprinting blog post to www
## Description

Migrate password blog post https://blog.mozilla.org/firefox/how-to-block-fingerprinting-with-firefox/ to www with updated copy.

Design Template: same as https://www.mozilla.org/en-US/firefox/browsers/windows-64-bit/

Copy: https://docs.google.com/document/d/19pCoqieDwUeT2BQ7YLRnp2VweKPgr8jjy787F0Ygi58/edit

Url: /firefox/features/block-fingerprinting/

Subnav like all /features/ pages

---

#### Success Criteria
- [x] Page is coded
- [ ] Blog article is redirected 301 to the new url on www
- [x] Add page to /more
- [x] Add page to /features
- [x] Link this page from /new (Block Protect you Privacy - Linktext "Fingerprinter Blocking")

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- add content and assets for:
    - features/fingerprinting
    - features/translate
    - privacy/safe-passwords
- add links on /more
- add links on /features
- add safe-passwords to privacy subnav

## Issue / Bugzilla link

Fix #9761
Fix #9759
Fix #9760

## Testing

- http://localhost:8000/en-US/firefox/features/translate/
- http://localhost:8000/en-US/firefox/features/fingerprinting/
- http://localhost:8000/en-US/firefox/privacy/safe-passwords/
- http://localhost:8000/en-US/firefox/features/
- http://localhost:8000/en-US/firefox/more/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/features/translate.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <p class="mzp-c-article-intro">{{ ftl('features-translate-the-internet-is', attrs='href="%s"'|safe|format(url('firefox.all'))) }}</p>
+
+    <h2>{{ ftl('features-translate-to-google-translate') }}</h2>
+    <p>{{ ftl('features-translate-google-translate-with', attrs='href="https://addons.mozilla.org/en-US/firefox/addon/to-google-translate/" rel="external noopener"') }}</p>
+
+
+    <p><a class="mzp-c-button mzp-t-product" href="https://addons.mozilla.org/firefox/addon/to-google-translate/" rel="external noopener">{{ ftl('features-translate-get-the-extension') }}</a></p>
+    <p>{{ ftl('features-translate-once-installed-simply') }}</p>
+
+    <img src="{{ static('img/firefox/features/translate/translation.png') }}">
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/translate.html:52]: Missing `alt` here, too.

**Review by craigcook — CHANGES_REQUESTED** [review]
Looks good, just some images missing `alt` attributes.

**Comment by stephaniehobson** [comment]
Updated!

**Review by craigcook — APPROVED** [review]
🔒

**Pull request merged**